src/vivarium_csu_alzheimers/constants/data_keys.py

```diff
@@ -38,6 +38,9 @@ class __Alzheimers(NamedTuple):
     # Prevalence will just be set to 1 so all simulants are created with the disease
     PREVALENCE: str = "cause.alzheimers_disease_and_other_dementias.prevalence"
     INCIDENCE_RATE: str = "cause.alzheimers_disease_and_other_dementias.incidence_rate"
+    TOTAL_POPULATION_INCIDENCE_RATE: str = (
```

Comment:
Why can't you just use the key `"cause.alzheimers_disease_and_other_dementias.raw_incidence_rate"` and `load_standard_data()`?